.gitignore

```diff
@@ -2,11 +2,11 @@
 /.php-cs-fixer.php
 /.phpunit.result.cache
 /box.json
+/compose.override.yaml
 /composer.lock
 /dev-tools/bin/
 /dev-tools/phpstan/cache/
 /dev-tools/vendor/
-/docker-compose.override.yaml
```

Comment:
overall OK, but I guess few ppl will get confused for 2 seconds seeing the file start popping up as non-ignored in their local repo

Reply:
Yes, I thought about adding new entry in `.gitignore` and leaving the old one, but in the end I decide to remove it because it will help to spot the change. File will be there for committing, causing questions. If it still would be ignored, then Docker commands could work wrong but developer wouldn't knew why.